src/utils/hooks/api/vaults/get-vault-data.ts

```diff
@@ -90,10 +91,10 @@ const getVaultData = async (vault: VaultExt, accountId: AccountId, prices: Price
     accountId,
     vault.backingCollateral.currency
   );
-  const issuableTokens = await window.bridge.vaults.getIssuableTokensFromVault(
-    accountId,
-    vault.backingCollateral.currency
-  );
+  const issuableTokens =
+    vaultExt.status === VaultStatusExt.Liquidated
+      ? newMonetaryAmount(0, vault.backingCollateral.currency)
+      : await window.bridge.vaults.getIssuableTokensFromVault(accountId, vault.backingCollateral.currency);
```

Comment:
I think it would be clearer to catch the error and only return zero when the error relates to a liquidated vault. That way it might be easier for the reader to understand where the default zero comes from? Up to you though

Reply:
> I think it would be clearer to catch the error and only return zero when the error relates to a liquidated vault. That way it might be easier for the reader to understand where the default zero comes from? Up to you though

Yes, tend to agree with you here - I'll add a comment to change this later. It's a bit of a shift in how we handle errors so would want to put a bit of thought into it (and maybe bring it in alongside some error changes in the lib)